back-end/tmp.py

```diff
@@ -0,0 +1,36 @@
+from objects.RModelWrapper import RModelWrapper
```

Comment:
This file seems to be for debugging only. We can delele it if it's not used anywhere else. Where does it come from though?